homeassistant/components/edimax/switch.py

```diff
@@ -43,6 +45,25 @@ def __init__(self, smartplug, name):
         self._now_power = None
         self._now_energy_day = None
         self._state = False
+        self._supports_power_monitoring = False
+        self._info = None
+        self._mac = None
+
+    @property
+    def unique_id(self):
+        """Return the device's MAC address."""
+        return self._mac
+
+    @property
+    def device_info(self):
```

Comment:
Device info requires a config flow and config entry.

Please remove `device_info` for now.